### PR TITLE
Update snagit.rb

### PR DESCRIPTION
**Pull request by My2ndAngelic**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/snagit.rb [hunk 1]
```diff
@@ -1,8 +1,34 @@
 cask 'snagit' do
-  version :latest
-  sha256 :no_check
+  if MacOS.release == :leopard
+    version '1.0.4'
+    sha256 :no_check
+    url 'http://download.techsmith.com/snagitmac/enu/104/snagit.dmg'
```
Comment (vitorgalvao) [inline Casks/snagit.rb]: Use `url "http://download.techsmith.com/snagitmac/enu/#{version.no_dots}/snagit.dmg"` instead. Same thing for all the others.

**Comment by vitorgalvao** [comment]
I was under the impression I had commented this, but apparently not. When `version` is present, you also need to set `sha256`. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/snagit.rb [hunk 2]
```diff
@@ -1,8 +1,30 @@
 cask 'snagit' do
-  version :latest
-  sha256 :no_check
+  if MacOS.release == :leopard
+    version '1.0.4'
+    sha256 'bfefc1c351485bffa01b7961685120306eb7624ec3ef63ba2b6d8049c846cd07'
+    url "http://download.techsmith.com/snagitmac/enu/#{version.no_dots}/snagit.dmg"
+  elsif MacOS.release == :snow_leopard || MacOS.release == :lion
```
Comment (jawshooah) [inline Casks/snagit.rb]: `MacOS.release <=  :lion`

**Comment by My2ndAngelic** [comment]
I have updated the condition

**Comment by jawshooah** [comment]
Thanks!

**Pull request merged**